pkg/controller/machine/actuator.go

```diff
@@ -18,13 +18,14 @@ package machine
 
 import (
 	clusterv1 "sigs.k8s.io/cluster-api/pkg/apis/cluster/v1alpha1"
+	"sigs.k8s.io/cluster-api/pkg/cert"
 )
 
 // Actuator controls machines on a specific infrastructure. All
 // methods should be idempotent unless otherwise specified.
 type Actuator interface {
 	// Create the machine.
-	Create(*clusterv1.Cluster, *clusterv1.Machine) error
+	Create(*clusterv1.Cluster, *clusterv1.Machine, *cert.CertificateAuthority) error
```

Comment:
I don't this we should modify this interface until we have a standard way of specifying CA input data. Making this change breaks any current actuator.

I think it would be better to make this a field of the GCE actuator rather than a parameter for the time being. Long term this would be cluster level data, and it'll be part of the first parameter in this function.

Reply:
Got it and agree -- will back this out and pass it into the machineactuator.